rpc/oauth/web_oauth.go

```diff
@@ -128,14 +129,15 @@ func SignWebAuthAccessToken(key *rsa.PrivateKey, entity, aud, iss, keyID string)
 		},
 		Claims: rpc.JWTClaims{
 			RegisteredClaims: jwt.RegisteredClaims{
+				ID:       uuid.NewString(),
```

Comment:
@AdamMagaluk does auth0 provide the jti? If not, any suggestions on how to work around us wanting unique identification of JWTs?

Reply:
See my other comment.